codalab/model/worker_model.py

```diff
@@ -451,3 +456,36 @@ def has_reply_permission(self, user_id, worker_id, socket_id):
             if row:
                 return True
             return False
+
+    def send_json(self, data: dict, worker_id: str, timeout_secs: int = 60):
```

Comment:
The new function that sends data through websockets.

The other functions (send_json_message_with_sock, send_stream) send data through the AF Unix sockets. (We still need those since file data is still send through AF Unix sockets and both those functions are used for file data. send_json_message_with_sock is also required for files since there's a [header message](https://github.com/codalab/codalab-worksheets/blob/master/codalab/rest/workers.py#L118-L128) pertaining to the file.)